cime_config/SystemTests/lilacsmoke.py

```diff
@@ -407,6 +407,7 @@ def run_phase(self):
         # case.get_mpirun_cmd creates a command that runs the executable given by
         # case.run_exe. So it's important that (elsewhere in this test script) we create a
         # link pointing from that to the atm_driver.exe executable.
+        self._case.load_env(reset=True)
```

Comment:
Since, this became a whole thing that took a lot of time to solve, I suggest adding a comment regarding it.

Reply:
Thank you @ekluzek 
Let me know if you would modify the comment that I added.